scripts/plop/templates/component/story.tsx.hbs

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { {{pascalCase name}} } from './{{pascalCase name}}';
+
+const stories = storiesOf('{{pascalCase name}}', module);
+
+stories.add('Story', () => (
```

Comment:
Can we swap this out to the new storybook api?

see https://storybook.js.org/docs/basics/writing-stories/#basic-story, which does not use the `stories.add` api

Reply:
For some reason couldn't get it to work. Let's scope this in another PR and update all stories to use the new [Component Story Format (CSF)](https://storybook.js.org/docs/formats/component-story-format/). 

Issue created: https://github.com/lightspeed/flame/issues/54